guides/testing-rest-api-integrations-using-mockserver/testing-rest-api-integrations-using-mockserver.adoc

```diff
@@ -0,0 +1,221 @@
+common:header-top.adoc[]
+
+== Getting Started
+
+In this guide, you will learn how to
+
+* Create a Micronaut application that talks to an external REST API.
+* Test the external API integration using https://testcontainers.com/modules/wiremock/[Testcontainers WireMock module].
+
+
+common:requirements.adoc[]
+* https://www.testcontainers.org/supported_docker_environment/[A Docker environment supported by Testcontainers]
+
+common:completesolution.adoc[]
+
+== What we are going to achieve in this guide
+We will create a Micronaut application that talks to an external REST API.
+Then, we will test the external REST API integration using
+the https://testcontainers.com/modules/wiremock/[Testcontainers WireMock module].
+
+common:create-app-features.adoc[]
+
+== WireMock dependency
+
+Once the application is generated, add the *WireMock* library as a test dependency.
+
+dependency:mockserver[groupId=org.testcontainers,scope=test]
```

Comment:
This is MockServer dependency. This should be changed to WireMock dependency.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-guides/pull/1330/commits/296eb6593a7dd3dc8cd492f0a0869f448e5d7e39